src/processor.ts

```diff
@@ -170,23 +171,25 @@ processor.run(new TypeormDatabase(), async ctx => {
     scoresMap.forEach(async function(userScores, accountId) {
         let earlyBirdSpecialScore = userScores.earlyBirdSpecialScore
         if (earlyBirdSpecialScore != null) {
-            console.log('persisting userScores', earlyBirdSpecialScore)
+            console.log('persisting userScores for EarlyBirdSpecial game', earlyBirdSpecialScore)
             await ctx.store.save(earlyBirdSpecialScore)
```

Comment:
Quick question about this.
Is may sound counter intuitive, or I may have misunderstood what's going on. But is it possible to add all `earlyBirdSpecialScore` (and same for the other scores) to a cumulative List and then save the entire list at the bottom?

This should increase the overall performance, doing a batch save, instead of multiple ones in a loop for every user.

Again: unless I misunderstood something

Reply:
You're quite right, I can save some DB roundtrips this way. Pls see the update